### PR TITLE
Configure GA and enable tag manager

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@miltontony Ready for review cc @peterp

File: html/index.html [hunk 1]
```diff
@@ -15,6 +15,15 @@
...
+<!-- Google Tag Manager -->
+<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-K64T4Z"
+height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','GTM-K64T4Z');</script>
+<!-- End Google Tag Manager -->
 <body class="index">
```
Comment (miltontony) [inline html/index.html]: Can you please put this inside the body?

**Comment by g1039** [comment]
@miltontony Ready for re-review

**Comment by miltontony** [comment]
👍

**Pull request merged**